### PR TITLE
Improve CSV/XLSX output for seller responses

### PR DESCRIPTION
**Pull request by maxious**

### CODE REVIEW

File: app/buyers/views/buyers.py [hunk 1]
```diff
@@ -307,6 +308,9 @@ def prepared_response_contents_for_brief(brief, responses):
     ess_req_names = brief.get(ESS, [])
     nth_req_names = brief.get(NTH, [])
 
+    def csv_cell_sanitize(text):
+        return re.sub(r"^(=|\+|-|@|!|\|)+", '', text.strip())
```
Comment (djrobstep) [inline app/buyers/views/buyers.py]: does xlsxwriter have some existing functions for doing this? might be more robust to use that than a random regex

File: app/buyers/views/buyers.py [hunk 2]
```diff
@@ -319,6 +323,10 @@ def row(r):
         answers.update({'Day rate': r.get('dayRate', '')})
         answers.update(zip(ess_req_names, ess_responses))
         answers.update(zip(nth_req_names, nth_responses))
+
+        for q in answers.keys():
+            answers[q] = csv_cell_sanitize(answers[q])
```
Comment (djrobstep) [inline app/buyers/views/buyers.py]: this is more succinct as: 

`answers = { k: csv_cell_sanitize(v) for k, v in answers.items() }`

Comment (maxious) [inline app/buyers/views/buyers.py]: This loses the ordering of the OrderedDict though.

File: app/buyers/views/buyers.py (hunk 1 above)
Comment (maxious) [inline app/buyers/views/buyers.py]: They seem to encourage use of formulas, I found some more characters to remove in xlsxwriter but no helper ;)

File: app/buyers/views/buyers.py (hunk 2 above)
Comment (djrobstep) [inline app/buyers/views/buyers.py]: Ah right, didn't spot the od. Still worth using `items()` though:

`for k, v in answers.items():`

File: app/buyers/views/buyers.py [hunk 3]
```diff
@@ -307,6 +308,9 @@ def prepared_response_contents_for_brief(brief, responses):
     ess_req_names = brief.get(ESS, [])
     nth_req_names = brief.get(NTH, [])
 
+    def csv_cell_sanitize(text):
+        return re.sub(r"^(=|\+|-|@|!|\|{|}|\[|\]|<)+", '', unicode(text).strip())
+
```
Comment (djrobstep) [inline app/buyers/views/buyers.py]: What's the purpose of the `unicode` here?

Comment (djrobstep) [inline app/buyers/views/buyers.py]: The responses should already be coming back as unicode so is it just for the day rate default response?

Comment (maxious) [inline app/buyers/views/buyers.py]: To handle bool type objects which implicitly were converted to string by the csv writer

**Pull request merged**